packages/iov-bns/src/bnsconnection.posttxs.spec.ts

```diff
@@ -603,6 +612,275 @@ describe("BnsConnection (post txs)", () => {
+      const signed2 = await profile.signTransaction(identity, renewDomainTx, bnsCodec, nonce2);
+      const response2 = await connection.postTx(bnsCodec.bytesToPost(signed2));
+      const blockInfo2 = await response2.blockInfo.waitFor(info => !isBlockInfoPending(info));
+      expect(blockInfo2.state).toEqual(TransactionState.Succeeded);
+
+      await tendermintSearchIndexUpdated();
+
+      // Find renew transaction
+      const searchResult2 = (await connection.searchTx({ signedBy: address })).filter(
+        isConfirmedAndSignedTransaction,
+      );
+      expect(searchResult2.length).toEqual(2);
+      const firstSearchResultTransaction2 = searchResult2[1].transaction;
+      assert(isRenewDomainTx(firstSearchResultTransaction2), "Expected RenewDomainTx");
+
+      connection.disconnect();
+    });
+
+    it("can delete a domain", async () => {
+      pendingWithoutBnsd();
+      const connection = await BnsConnection.establish(bnsdTendermintUrl);
+      const registryChainId = connection.chainId();
+
+      const profile = new UserProfile();
+      const wallet = profile.addWallet(Ed25519HdWallet.fromEntropy(Random.getBytes(32)));
+      const identity = await profile.createIdentity(wallet.id, registryChainId, HdPaths.iov(0));
+
+      // we need funds to pay the fees
+      const address = identityToAddress(identity);
+      await sendTokensFromFaucet(connection, address, registerAmount);
+
+      // Create and send registration
+      const domain = randomDomain();
+      const registerDomainTx = await connection.withDefaultFee<RegisterDomainTx>(
+        {
+          kind: "bns/register_domain",
+          chainId: registryChainId,
+          domain,
+          admin: address,
+          hasSuperuser: true,
+          msgFees: [],
+          accountRenew: 100,
+        },
+        address,
+      );
+      const nonce = await connection.getNonce({ pubkey: identity.pubkey });
+      const signed = await profile.signTransaction(identity, registerDomainTx, bnsCodec, nonce);
+      const response = await connection.postTx(bnsCodec.bytesToPost(signed));
+      const blockInfo = await response.blockInfo.waitFor(info => !isBlockInfoPending(info));
+      expect(blockInfo.state).toEqual(TransactionState.Succeeded);
+
+      await tendermintSearchIndexUpdated();
+
+      // Find registration transaction
+      const searchResult = (await connection.searchTx({ signedBy: address })).filter(
+        isConfirmedAndSignedTransaction,
+      );
+      expect(searchResult.length).toEqual(1);
+      const firstSearchResultTransaction = searchResult[0].transaction;
+      assert(isRegisterDomainTx(firstSearchResultTransaction), "Expected RegisterDomainTx");
+      expect(firstSearchResultTransaction.admin).toEqual(address);
+
+      // Delete domain
+      const deleteDomainTx = await connection.withDefaultFee<DeleteDomainTx>(
+        {
+          kind: "bns/delete_domain",
+          chainId: registryChainId,
+          domain,
```

Comment:
This shorthand notation is disallowed in IOV Core by tslint rule `"object-literal-shorthand": [true, "never"],`. So it must be `domain: domain,`

Motivation is that an inconsistent literal with shorthands and non-shorthands is harder to read.